src/api/env.js

```diff
@@ -0,0 +1,3 @@
+const TEMPER_URL = "https://ii1302-group3-api.herokuapp.com/temperatures/current";
```

Comment:
Should we delete this file and make this file `gitignored` or should we have this flaw as this is only a school project.